unrarall

```diff
@@ -165,6 +167,40 @@ function getUnrarFlags()
   esac
 }
 
+#function to check if an archive have been extracted already
+function isAlreadyExtracted()
+{
+  case "$1" in
+    unrar)
+      contains=($(unrar lb "$2"))
```

Comment:
I don't have any rar files to hand to test this. Does `unrar lb` spit out the correct paths for where we extract them?